src/reducers/index.js

```diff
@@ -0,0 +1,42 @@
+const initMessage = { message: "Działa!", time: new Date(), users: [] }
+
+const reducer = (state = initMessage, action) => {
+    switch (action.type) {
+        case 'getCurrentTime':
+            return {
+                ...state,
+                time: new Date()
+            }
+        case 'addUser':
+            const { users } = state
+            let id
+            const usersLength = users.length
+
+            if (usersLength !== 0) {
+                id = users[usersLength - 1].id + 1
+            } else { id = 0 }
```

Comment:
Lepiej skorzystać z `uuid` - nie trzeba kombinować ;P

Reply:
hmm... ale wydaje mi się, że wtedy nie jest zgodne z treścią zadania ;) "Identyfikator ma być kolejnym największym numerem względem obecnie przechowywanych id."

Reply:
ok, w takim mój błąd - nie pamiętałem o tym :D

To najlepiej inaczej podejść do tego algorytmu. W tym rozwiązaniu nie jestem pewny jego poprawności. Jeśli ktoś doda parę elementów potem coś pousuwa to nie jestem pewny, czy zawsze ostatni element będzie miał największe id co może rodzić problem ;) Może dla pewności przeszukałbym całą tablicę np. tak: `const max = users.reduce((acc, curr) => curr.id > acc ? curr.id : acc, 0)`